cmd/release/docs/existingdocs/readme.go

```diff
@@ -1,10 +1,10 @@
-package existingdocs
```

Comment:
haha the name of this file, `readme.go`, broke my brain for a second; I was like 'it's a readme, but not markdown, and it's showing go code, what's happening....' 🤯